### PR TITLE
Origamify

### PR DESCRIPTION
**Pull request by matthew-andrews**
before:-

before

after:-

after

@keirog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,10 @@ Bower component for a quote.
 
 ## Types of quote
 
-A quote can be one of 3 types
+A quote can be one of 1 type
```
Comment (keirog) [inline README.md]: Funny but probably confusing to read

Comment (matthew-andrews) [inline README.md]: ok fixed

File: demos/standard.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<html class="o-hoverable-on ">
+<head>
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+	<title>n-quote: standard demo</title>
+	<meta name="viewport" content="initial-scale=1.0, width=device-width" />
+	<script src="//polyfill.webservices.ft.com/v1/polyfill.min.js?features=default"></script>
+	<style>body { margin: 0; } .demo-js .o--if-nojs { display: none; }</style>
+	<script>(function(d) { d.className = d.className + ' demo-js'; })(document.documentElement);</script>
+	<link rel="stylesheet" href="/bundles/css?modules=n-quote:/demos/src/demo.scss,o-fonts@^1.4.0" />
```
Comment (keirog) [inline demos/standard.html]: o-fonts@^2.0.0

Comment (matthew-andrews) [inline demos/standard.html]: i just removed that

Comment (matthew-andrews) [inline demos/standard.html]: it comes through next-sass-setup

Comment (keirog) [inline demos/standard.html]: ah ok, I didn't know if it even did anything--just auto-commented when I saw the old number

**Comment by keirog** [comment]
:+1:

File: Makefile [hunk 3]
```diff
@@ -4,11 +4,14 @@ clean:
...
 
 install:
-	npm install origami-build-tools@3.3.5
 	obt install
 
 test:
 	obt verify
 
+run:
+	obt demo --runServer --local
```
Comment (AlbertoElias) [inline Makefile]: runServer implies local

File: demos/base.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	<title>n-quote: base demo</title>
+	<meta name="viewport" content="initial-scale=1.0, width=device-width" />
+	<script src="//polyfill.webservices.ft.com/v1/polyfill.min.js?features=default"></script>
+	<style>body { margin: 0; } .demo-js .o--if-nojs { display: none; }</style>
+	<script>(function(d) { d.className = d.className + ' demo-js'; })(document.documentElement);</script>
+	<link rel="stylesheet" href="/bundles/css?modules=n-quote:/demos/src/demo.scss" />
+</head>
+<body>
+<h3>Default pull quote</h3>
+<blockquote class="n-quote">
```
Comment (AlbertoElias) [inline demos/base.html]: n-quote?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "origami-build-tools": "^4.2.0"
```
Comment (AlbertoElias) [inline package.json]: Oh, I like the idea of having this in our devDependencies, I think we should add this to all projects

Comment (matthew-andrews) [inline package.json]: i actually prefer your approach

File: demos/base.html (hunk 4 above)
Comment (matthew-andrews) [inline demos/base.html]: yeah needs renaming when it's properly properly origamified

File: Makefile (hunk 3 above)
Comment (matthew-andrews) [inline Makefile]: cool i didn't know that

Comment (matthew-andrews) [inline Makefile]: fixed in https://github.com/Financial-Times/n-quote/commit/ea9b6a707d2af0ca4613ecc833e9bc1ffb571f22

File: package.json (hunk 5 above)
Comment (matthew-andrews) [inline package.json]: [ because you have to have `npm install` in your `travis.yml` **anyway** ]

**Pull request merged**